spec/requests/confirmation_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Confirmations', type: :request do
+  let(:user) { create :user }
+
+  describe 'POST /create' do
+    it 're-sends confirmation' do
+      post '/api/auth/confirmation', params: { email: user.email }
+      expect(response).to have_http_status(:ok)
+    end
+  end
+end
```

Comment:
This is not a mistake, but it is good practice to end files with a newline (`\n`) character, and it stops the [annoying warnings](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log).